app/models/user.rb

```diff
@@ -107,6 +93,43 @@ def decorate
     UserDecorator.new(self)
   end
 
+  def max_login_attempts?
+    second_factor_attempts_count.to_i >= max_login_attempts
+  end
+
+  def max_login_attempts
+    3
```

Comment:
not a requirement for this PR, but I feel like this is a good candidate for an `AppConfig` setting or something more flexible in the future